### DISCUSSION TITLE
Are Vector Stores supported?

### DISCUSSION THREAD

**Discussion by mrceolla** [openai-php/client#387] [General]
Hi,

Thank you for this software.  I am very new to programming OpenAI assistants.  I am wanting to create a Vector Store for file search option, but I cannot find documentation mentioning anything about Vector Stores.  Am I missing it?  Are Vector Stores supported yet?

I'm referring to this:  https://platform.openai.com/docs/api-reference/vector-stores

**Comment by gehrisandro** [comment]
see https://github.com/openai-php/client/discussions/411

**Reply by mrceolla** [reply]
Fantastic!  Thank you!

**Comment by gehrisandro** [comment]
Released: https://github.com/openai-php/client/releases/tag/v0.10.1
